src/sentry/options/defaults.py

```diff
@@ -163,8 +163,8 @@
 # Skip nodestore save when saving an event
 register("store.save-event-skips-nodestore", default=False, flags=FLAG_PRIORITIZE_DISK)
 
-# Check postgres for duplicates before saving an event
-register("store.check-duplicates", default=True, flags=FLAG_PRIORITIZE_DISK)
+# Skip saving an event to postgres
+register("store.skip-pg-save", default=True, flags=FLAG_PRIORITIZE_DISK)
```

Comment:
default=True ?

Reply:
We should roll this out with this set to False in production. But I'd like the tests to run with the new behavior.